### PR TITLE
Reorganize transaction info

### PR DESCRIPTION
**Pull request by cshcomcom**
Resolve: #934

### CODE REVIEW

File: tx-pool/index.js [hunk 1]
```diff
@@ -465,6 +466,7 @@ class TransactionPool {
     const addrToTimestamp = {};
     for (const voteTx of block.last_votes) {
       const txTimestamp = voteTx.tx_body.timestamp;
+      const txExecutedAt = _.get(this.transactionTracker.get(voteTx.hash), 'executed_at', -1);
```
Comment (platfowner) [inline tx-pool/index.js:469]: minor) what about 'executedAt'?

File: tx-pool/index.js [hunk 2]
```diff
@@ -484,6 +487,7 @@ class TransactionPool {
       const tx = block.transactions[i];
       const txNonce = tx.tx_body.nonce;
       const txTimestamp = tx.tx_body.timestamp;
+      const txExecutedAt = _.get(this.transactionTracker.get(tx.hash), 'executed_at', -1);
```
Comment (platfowner) [inline tx-pool/index.js:490]: ditto?

File: tx-pool/index.js (hunk 1 above)
Comment (cshcomcom) [reply tx-pool/index.js]: Thanks. Done!

File: tx-pool/index.js (hunk 2 above)
Comment (cshcomcom) [reply tx-pool/index.js]: Done!

**Review by platfowner — APPROVED** [review]
LGTM
Thanks!

**Comment by platfowner** [comment]
Let me merge it for v1.0.5 release. @cshcomcom

**Pull request merged**